### ISSUE TITLE
[a11y] 9.2.1.1 - Usable without mouse

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5388]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
- **"header“-area / Buttons "Apps-Menü" and "Benutzermenü":** It is possible to activate both menus with the keyboard but they won’t close when leaving them with the keyboard (tabbing). In case of „Benutzermenü" this leads to background elements getting a keyboard focus which is covered by the menu.
**Similar issue** with the button "Neu" on page 1.
- **Pages 1 and 2:** The date picker dialogue ("Öffentlicher Link") is not operable with the keyboard. Also it is not possible to enter a date in the field.
**Note:** When the date picker will be optimised, labels should be in German language.

## Status: rather fulfilled
- **Page 3 (Gelöschte Dateien)**
See general notes
- **Page 4 (Konto)**
See general notes

## Status: partly fulfilled
- **Page 1 (Alle Dateien)**
See general notes
- **Page 2 (Mit anderen geteilten Dateien)**
See general notes

**Comment by pascalwengerter** [comment]
Needs a discussion if we _always_ want to close tooltips once they loose focus (by adding sth along the lines of https://atomiks.github.io/tippyjs/v6/plugins/#hideonpopperblur to the ODS)

**Comment by stevefrenzel** [comment]
> Needs a discussion if we _always_ want to close tooltips once they loose focus (by adding sth along the lines of https://atomiks.github.io/tippyjs/v6/plugins/#hideonpopperblur to the ODS)

@tbsbdr @kulmann @pascalwengerter 

What's the status of this discussion? Also would like to mention https://duetds.github.io/date-picker/ as accessible date picker alternative, as the current one is... not good regarding accessibility.

**Comment by pascalwengerter** [comment]
Superseded by the more recent #10730
